lib/webapp.js

```diff
@@ -187,6 +192,32 @@ function createDataserviceStorage(pluginId) {
   })
 }
 
+function checkNodeSupportVersion(logger) {
+  var currentDate = new Date();
+
+  for (var majorVer in nodeSupportTimeTable) {
+    var months = nodeSupportTimeTable[majorVer].getMonth() - currentDate.getMonth() + 
+      (12 * (nodeSupportTimeTable[majorVer].getFullYear() - currentDate.getFullYear()));
+
+    if (parseInt(nodeMajorVer) <= majorVer) {
+      if (currentDate > nodeSupportTimeTable[majorVer]) // We use break so if version is multiple iterations old, we don't get multiple messages
+      {
+        logger.warn("YOUR NODE VERSION IS OUT OF DATE. ZOWE IS NO LONGER SUPPORTING NODE VERSION " + majorVer + ".");
+        break;
+      } else if (months >= 6) {
```

Comment:
This seems like unnecessarily verbose logging that may need to be cleaned up later when its irrelevant (after node version 6 is unsupported)